### PR TITLE
Feature/49964 regression test containers Part #1

### PR DESCRIPTION
**Pull request by GuyHarwood**
Part 1 of 2.

- containerise service via docker compose

### CODE REVIEW

File: deploy/regression-test-container/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    command: ["./docker-start-dev.sh"]
+  spa:
+    build:
+      context: ../../pupil-spa
+    container_name: mtc_spa
+    ports:
+      - 4200:80
+    depends_on:
+      - pupil_api
+#    command: ["./docker-start-dev.sh"]
```
Comment (jon-shipley) [inline deploy/regression-test-container/docker-compose.yml:113]: Is this commented out command still needed?

**Review by jon-shipley — APPROVED** [review]
LGTM 💯

File: deploy/regression-test-container/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,113 @@
+version: '3'
+services:
+  sqldb:
+    image: "mcr.microsoft.com/mssql/server:2019-CU6-ubuntu-16.04"
+    container_name: mtc_sql
+    ports:
+      - "1433:1433"
+    environment:
+      SA_PASSWORD: "Mtc-D3v.5ql_S3rv3r"
```
[the remark this answers is not in the document]
Comment (GuyHarwood) [reply deploy/regression-test-container/docker-compose.yml:9]: probably no need?  this is the default value used for local development and can be the same for an isolated test run

File: deploy/regression-test-container/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,112 @@
+version: '3'
+services:
+  sqldb:
+    image: "mcr.microsoft.com/mssql/server:2019-CU6-ubuntu-16.04"
```
Comment (jon-shipley) [inline deploy/regression-test-container/docker-compose.yml:4]: This sql version is getting out of date.  The latest is CU14 : https://hub.docker.com/_/microsoft-mssql-server

Comment (GuyHarwood) [reply deploy/regression-test-container/docker-compose.yml:4]: ive created ticket 50650 to cover the work

**Pull request merged**